service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -89,4 +90,13 @@ private void run1335(CaseDetails caseDetails) {
         caseDetails.getData().putAll(migrateCaseService.removeSkeletonArgument(getCaseData(caseDetails),
             "e4e70bf5-4905-4c13-9d59-d20a202b6c9a", migrationId));
     }
+
+    private void run1303(CaseDetails caseDetails) {
+        var migrationId = "DFPL-1303";
+        var possibleCaseIds = List.of(1652697388556674L);
+        migrateCaseService.doCaseIdCheckList(caseDetails.getId(), possibleCaseIds, migrationId);
+        caseDetails.getData().putAll(migrateCaseService.removeApplicationDocument(getCaseData(caseDetails),
+            migrationId,
+            UUID.fromString("7b381f49-d6f9-4a17-a72a-5e39fb48a67")));
```

Comment:
should be 7b381f49-d6f9-4a17-a72a-5e39fb48a671?

Reply:
updated! nice spot!!